src/assets/index.html

```diff
@@ -1,17 +1,28 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>My App</title>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>My App</title>
 
-  <style>
-    /* remove preseted padding */
-    body { margin: 0px }
-  </style>
-</head>
-<body>
-  <div id="root"></div>
-</body>
+    <style>
+      /* remove preseted padding */
+      body { margin: 0px }
+    </style>
+  </head>
+
+  <body>
+    <div id="root"></div>
+  </body>
+
+  <script>
+    // Check that service workers are registered
```

Comment:
Should be moved to `app/index.js`. Let's try to import this script and then register it when the app starts